src/test/java/net/sf/jabref/importer/fileformat/IsiImporterTest.java

```diff
@@ -273,97 +342,167 @@ public void testGetIsCustomImporter() {
+
+            Assert.assertEquals("In this letter, the problem of detecting urban road "
+                    + "networks from high-resolution optical/synthetic aperture "
+                    + "radar (SAR) images is addressed. To this end, this letter "
+                    + "exploits a priori knowledge about road direction "
+                    + "distribution in urban areas. In particular, this letter "
+                    + "presents an adaptive filtering procedure able to capture the "
+                    + "predominant directions of these roads and enhance the "
+                    + "extraction results. After road element extraction, to both "
+                    + "discard redundant segments and avoid gaps, a special "
+                    + "perceptual grouping algorithm is devised, exploiting "
+                    + "colinearity as well as proximity concepts. Finally, the road "
+                    + "network topology is considered, checking for road "
+                    + "intersections and regularizing the overall patterns using "
+                    + "these focal points. The proposed procedure was tested on a "
+                    + "pair of very high resolution images, one from an optical "
+                    + "sensor and one from a SAR sensor. The experiments show an "
+                    + "increase in both the completeness and the quality indexes " + "for the extracted road network.",
+                    a.getField("abstract"));
+
+        }
+    }
+
+    @Test
+    public void testIEEEImport() throws IOException {
+        IsiImporter importer = new IsiImporter();
+
+        try (InputStream is = IsiImporterTest.class.getResourceAsStream("IEEEImport1.txt")) {
+
+            List<BibEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+
+            Assert.assertEquals(1, entries.size());
+            BibEntry a = entries.get(0);
+
+            Assert.assertEquals(a.getType().getName(), BibtexEntryTypes.ARTICLE, a.getType());
+            Assert.assertEquals("Geoscience and Remote Sensing Letters, IEEE", a.getField("journal"));
+            Assert.assertEquals(
+                    "Improving Urban Road Extraction in High-Resolution Images Exploiting Directional Filtering, Perceptual Grouping, and Simple Topological Concepts",
+                    a.getField("title"));
+
+            Assert.assertEquals("4", a.getField("volume"));
+            Assert.assertEquals("3", a.getField("number"));
+
+            Assert.assertEquals("1545-598X", a.getField("SN"));
 
+            Assert.assertEquals("387--391", a.getField("pages"));
+
+            Assert.assertEquals("Gamba, P. and Dell'Acqua, F. and Lisini, G.", a.getField("author"));
+
+            Assert.assertEquals("2006", a.getField("year"));
+
+            Assert.assertEquals("Perceptual grouping, street extraction, urban remote sensing", a.getField("keywords"));
+
```

Comment:
Try to use empty lines to group statements together. Like in a book where a paragraph groups together sentences.